### ISSUE TITLE
Editor scrolling not working on held down cursor up/down

### ISSUE THREAD

**Issue by jojastahl** [notepad-plus-plus/notepad-plus-plus#7719]
### Description of the Issue

The text in the editor is not scrolling when moving the cursor up or down below the window limits, if the cursor key is continuously hold down.

I just updated to the latest version (7.8.1) and got this behavior. I regret that I cannot tell the version I used before, where this worked.

### Steps to Reproduce the Issue

1. Open a larger document
2. Press and hold the cursor down key

### Expected Behavior

When the cursor reaches the bottom of the window, the text should scroll up and the lines the cursor is moving through should become visible.

### Actual Behavior

The cursor moves down, but when reaching the bottom of the window, it still moves but text does not scroll. The view will only be updated, when the cursor down key is released. While moving the cursor down, you can see the vertical scrollbar updating, but no text scroll.

### Debug Information

Notepad++ v7.8.1   (32-bit)
Build time : Oct 27 2019 - 22:46:07
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1903
OS Build : 18362.476
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by shawnkhall** [comment]
This issue persists in 7.8.2 on Win10 v1909.

**Comment by pateldeep2509** [comment]
Please share your notepad file in which the bug is reported.

**Comment by shawnkhall** [comment]
This file demonstrates the problem. 
[scroll-error.txt](https://github.com/notepad-plus-plus/notepad-plus-plus/files/3948582/scroll-error.txt)

Each line of the 250-line (3818 bytes) file has the line number in four tab-separated columns.

Open it in N++. Press AND HOLD the down arrow. You'll quickly notice that the active line no longer has number changing.

**Comment by sasumner** [comment]
@shawnkhall 

N++ seems to be working fine for me with this file...

Can you make an animated GIF and post it here to show exactly the bad behavior it exhibits for you?

**Comment by shawnkhall** [comment]
It looks like it only causes problems when the "Elastic Tabstops" plugin is enabled.

This did not happen prior to N++ 7.8.

**Comment by sasumner** [comment]
Ah, but @shawnkhall , is it the same as OP @jojastahl 's complaint, as the Elastic Tabstops plugin isn't in his Debug Info?  (Of course you may not know the answer to that)

As Elastic Tabstops plugin is a creation of @dail8859 's, maybe he has some insight into this issue?

**Comment by dail8859** [comment]
@shawnkhall I'm assuming this is related to Notepad++ updating the Scintilla version. Feel free to open an issue with specific details at https://github.com/dail8859/ElasticTabstops/issues

**Comment by jojastahl** [comment]
But what about my situation? I do not use ElasticTabstops, at least as far as I know.

**Comment by sasumner** [comment]
@jojastahl My earlier comment to @shawnkhall could have applied to you as well:

"Can you make an animated GIF and post it here to show exactly the bad behavior it exhibits for you?"

**Comment by jojastahl** [comment]
This shows the bug.
It seems to be file, type, size or whatever dependent, as I can't reproduce with all files.

![ScrollBug](https://user-images.githubusercontent.com/15796520/70904977-017a3b80-2003-11ea-8760-6654e1bf1fb9.gif)

**Comment by jojastahl** [comment]
This is the files that shows the behavior.

[LT-PLS-5.ISERVER-Database-[init-server_v7].log.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/3967862/LT-PLS-5.ISERVER-Database-.init-server_v7.log.zip)

**Comment by Ekopalypse** [comment]
Wonder if DSpellCheck plugin is causing this

**Comment by xylographe** [comment]
I tried that with French dictionary (at least 4 spelling mistakes per line). Scrolling becomes slower, but remains continuous.

**Comment by jojastahl** [comment]
@Ekopalypse: I disabled "Spell Check Document Automatically" and it works. But I didn't remark such behavior in the past. Is it a DSpellCheck bug?

**Comment by jojastahl** [comment]
I was using version 1.4.6 of DSpellCheck Plugin. Updated to 1.4.14: same behavior.

**Comment by Ekopalypse** [comment]
>Is it a DSpellCheck bug?

If disabling the "Spell Check Document Automatically" solved the issue then it is most likely a DSpellCheck bug. As plugins, normally, interact with npp and scintilla it might be an issue because of an combination of different things. Personally I would say, open an issue at DSpellCheck and provide, link to, the data to replicate the issue. The DSpellCheck dev might have the best insight view of what is happening with the plugin.

**Comment by xylographe** [comment]
See issue https://github.com/Predelnik/DSpellCheck/issues/195

Try [the latest release](https://github.com/Predelnik/DSpellCheck/releases/latest)

**Comment by jojastahl** [comment]
I confirm that DSpellCheck 1.4.15 resolves my issue. However, I had to download it from the DSpellCheck project because it was not shown as available update in the plugin manager.

**Comment by xylographe** [comment]
Unfortunately, Plugin Manager has been replaced by Plugin Admin. For plugins hosted on GitHub or SourceForge you can instruct GitHub/SourceForge to keep you informed about new releases.

**Comment by KDMcMullan** [comment]
I also have this issue. It's new as of my update to 7.8.2. I've no idea what version I previously had.
I've uninstalled all my plugins and the problem has gone away.

- I've reinstalled NppFTP 0.30.2 (previously 0.28.3) and the problem has NOT returned.
- I've reinstalled Hex-Editor (0.9.5 (same version as before). No problem.
- Reinstalled Pythion Indent 1.0.0.1 (same version as before). No problem.
- DSpellCheck 1.4.15 (previously 1.4.6). No problem.

That's all the plugins I'd had installed, suggesting that the problem is with Plugin Admin and offering a workaround: uninstall and reinstall plugins.
